tests/unit/mixins/test_mixin_methods.py

```diff
@@ -45,9 +45,47 @@ class M(GetMixin, FakeManager):
     assert isinstance(obj, FakeObject)
     assert obj.foo == "bar"
     assert obj.id == 42
+    assert obj._lazy is False
     assert responses.assert_call_count(url, 1) is True
 
 
+def test_get_mixin_lazy(gl):
+    class M(GetMixin, FakeManager):
+        pass
+
+    url = "http://localhost/api/v4/tests/42"
+
+    mgr = M(gl)
+    with responses.RequestsMock(assert_all_requests_are_fired=False) as rsps:
+        rsps.add(
+            method=responses.GET,
+            url=url,
+            json={"id": 42, "foo": "bar"},
+            status=200,
+            match=[responses.matchers.query_param_matcher({})],
+        )
+        obj = mgr.get(42, lazy=True)
+    assert isinstance(obj, FakeObject)
+    assert not hasattr(obj, "foo")
+    assert obj.id == 42
+    assert obj._lazy is True
+    # a `lazy` get does not make a network request
+    assert len(rsps.calls) == 0
+
+
+def test_get_mixin_lazy_missing_attribute(gl):
+    class FakeGetManager(GetMixin, FakeManager):
+        pass
+
+    manager = FakeGetManager(gl)
+    obj = manager.get(1, lazy=True)
+    assert obj.id == 1
+    with pytest.raises(AttributeError) as exc:
+        obj.missing_attribute
+    assert "as a `lazy` object" in str(exc)
+    assert "'missing_attribute'" in str(exc)
```

Comment:
The issue I had is that with the `textwrap.fill()` things move around. So the message can have a line-feed in the text. So that makes it difficult to use in a `match=`.

Reply:
Since we "theoretically" don't know where it can be split the regex would need to have a `.*` between every word and also not sure if the regex will work across multiple lines in the `match=`